src/layers/core/point-density-hexagon-layer/point-density-hexagon-layer.js

```diff
@@ -64,14 +66,38 @@ export default class PointDensityHexagonLayer extends Layer {
       props.radius = defaultRadius;
     }
 
+    if (Number.isFinite(props.upperPercentile) &&
+      (props.upperPercentile > 100 || props.upperPercentile < 0)) {
+      log.once(0, 'PointDensityHexagonLayer: upperPercentile should be between 0 and 100. ' +
+        'Assign to 100 by default');
+
+      props.upperPercentile = defaultUpperPercentile;
+    }
+
+    if (Number.isFinite(props.lowerPercentile) &&
+      (props.lowerPercentile > 100 || props.lowerPercentile < 0)) {
+      log.once(0, 'PointDensityHexagonLayer: lowerPercentile should be between 0 and 100. ' +
+        'Assign to 0 by default');
+
+      props.lowerPercentile = defaultLowerPercentile;
+    }
+
+    if (props.lowerPercentile >= props.upperPercentile) {
+      log.once(0, 'PointDensityHexagonLayer: lowerPercentile should not be bigger than ' +
+        'upperPercentile. Assign to 0 by default');
+
+      props.lowerPercentile = defaultLowerPercentile;
+    }
+
     super(props);
   }
 
   initializeState() {
     this.state = {
       hexagons: [],
-      countRange: null,
-      pickedCell: null
+      pickedCell: null,
+      sortedCounts: null,
+      countDomainByPercentile: null
```

Comment:
Naming of this variable seems long - can it be shortened? is there any other domain than count?

Would something simple like just `domain` or `valueDomain` work?